docs/_docs/plugins.md

```diff
@@ -722,6 +725,28 @@ The complete list of available hooks is below:
 
 ## Available Plugins
 
+### Gist
+
+The `gist` plugin is an example of tag plugin. To use it you'll need to add the
```

Comment:
I would just drop this. There's already an example of a tag plugin here: https://jekyllrb.com/docs/plugins/#tags

If people want to know how to use a plugin, they can go check the plugin's README.